scripts/merge-static-libs.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```
This makes the script adapted to a wider set of OS, where the BASH binary is not always located in the `/bin/bash` path.

Reply:
> `#!/usr/bin/env bash`

Okay... Though, I guess we'd need to update other places as well to ensure that Bash from PATH is used instead of `/bin/bash`.